CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.0] - pending
+
+### Added:
```

Comment:
Let's add a note about the new Kconfig option to enable CIDs: https://github.com/golioth/golioth-firmware-sdk/commit/0c46393bcd8e53e980c7203766f2636f7699576a

Reply:
Ah, I had some uncommitted changes to the changelog and also added the symbol related to CID. Thanks for catching this.